app/controllers/build_estimates_controller.rb

```diff
@@ -8,7 +8,8 @@ def update
       session_data.merge!(@form.session_attributes)
       estimate = load_estimate
 
-      next_step = StepsHelper.next_step_for(estimate, step)
+      next_step = estimate.asylum_support ? false : StepsHelper.next_step_for(estimate, step)
```

Comment:
Trying to add this logic here causes some weird side effects. For example, if you click yes to asylum support, then click the back button a few times to get you back to the first question in the flow, then try to go forwards again, this logic runs when trying to work out the next screen, so you get jumped to the check answers screen wrongly.

The "proper" way of doing this, which I think is the way we have to do it, is to modify each of the Section classes (apart from the CaseDetailsSection) to return an empty array in `steps_for(estimate)` if `estimate.asylum_support` is true. Much like we do for most sections if `estimate.passporting` is true.